config.yaml

```diff
@@ -1 +1,14 @@
-dynamo-endpoint: dynamodb.us-east-1.amazonaws.com
\ No newline at end of file
+dynamo-endpoint: http://localhost:8000
+
+# Approved Authentication Credentials
+approvedKeys:
```

Comment:
I know you followed what I put in my suggestion last time, but I think just as a style thing this should be `approved-keys` to match up with `dynamo-endpoint`.